__docs__/wonder-blocks-tokens/tokens-sizing.mdx

```diff
@@ -0,0 +1,71 @@
+import {Meta, Source} from "@storybook/blocks";
+
+import TokenTable from "../components/token-table";
+
+import {View} from "@khanacademy/wonder-blocks-core";
+import * as tokens from "@khanacademy/wonder-blocks-tokens";
+
+import ComponentInfo from "../components/component-info";
+import packageConfig from "../../packages/wonder-blocks-tokens/package.json";
+
+<Meta title="Packages / Tokens / Sizing" />
+
+# Sizing
+
+<ComponentInfo name={packageConfig.name} version={packageConfig.version} />
+
+All the available sizing values that can be used for margin, padding, width,
+height, border-width, etc.
+
+## Usage
```

Comment:
Would it be worth mentioning that the units are include in the token value? This is different from the current spacing tokens which are the numeric value only! 

Kinda related, do we think that the rem numeric value would be used on it's own? I was thinking maybe if consumers need to calculate something (like total width minus border and padding or something), but that should still work with units if the css `calc` function is used. Let me know if you know of other examples!

Reply:
Sidenote: I like that the unit is included in the token! Then we can have:
```
style: {
  padding: `${sizing.size_050} ${sizing.size_100}`,
}
```
Instead of
```
style: {
  padding: `${sizing.size_050}rem ${sizing.size_100}rem`,
}
```

Reply:
I think keeping always the `rem` unit would be preferred. In your example, `calc` uses `rem` properly when making calculations.